ui/app/pages/settings/contact-list-tab/edit-contact/edit-contact.container.js

```diff
@@ -22,7 +22,7 @@ const mapStateToProps = (state, ownProps) => {
   const contact = getAddressBookEntry(state, address) || state.metamask.identities[address]
   const { memo, name } = contact || {}
 
-  const chainId = state.metamask.network
+  const { chainId } = state.metamask.provider
```

Comment:
Do we have any selectors for this?

Reply:
A quick grep seems to say we do not but we use this key in 3 places already (4 with this):

```
ui/app/components/app/dropdowns/network-dropdown.js:    provider: state.metamask.provider,
ui/app/pages/confirm-transaction/conf-tx.js:    provider: state.metamask.provider,
ui/app/pages/routes/routes.container.js:    provider: state.metamask.provider,
```

Reply:
I see two lines like this one. Were these missed instances of the new provider schema?